mergify/manifest.json

```diff
@@ -7,10 +7,17 @@
     "overview": "README.md#Overview",
     "configuration": "README.md#Setup",
     "support": "README.md#Support",
+    "uninstallation": "README.md#Uninstallation",
     "changelog": "CHANGELOG.md",
     "description": "Mergify merge queue statistics integration",
     "title": "Mergify",
-    "media": [],
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Mergify - Dashboard",
+        "image_url": "images/dashboard.png"
```

Comment:
I also don't think I saw this image added in the folder.

Reply:
It's in the pull request: https://github.com/DataDog/integrations-extras/pull/2487/files#diff-04b1da02e9be6cd6633a8cc87468f398e9b6c2935400f3db01e138f088cdd1f5

Reply:
Ah, I see, those were getting filtered out. Thanks for confirming.